site/snippets/particle.ts

```diff
@@ -0,0 +1,25 @@
+import { ParticleNetwork } from '@particle-network/auth';
+import { ParticleProvider } from '@particle-network/provider';
+import { createWalletClient, custom } from "viem";
+import { WalletClientSigner, SmartAccountSigner } from "@alchemy/aa-core";
+
+const particle = new ParticleNetwork({
+  projectId: process.env.REACT_APP_PROJECT_ID as string,
+  clientKey: process.env.REACT_APP_CLIENT_KEY as string,
+  appId: process.env.REACT_APP_APP_ID as string,
+  chainName: 'polygon',
```

Comment:
Can you add some notion about choosing the right FACTORY_CONTRACT_ADDRESS, ENTRY_POINT_CONTRACT_ADDRESS for specific chain specified here in ParticleNetwork signer?

Reply:
Sure, done